pkg-r/R/querychat.R

```diff
@@ -13,8 +13,15 @@
 #' @param data_description A string containing a data description for the chat model. We have found
 #'   that formatting the data description as a markdown bulleted list works best.
 #' @param extra_instructions A string containing extra instructions for the chat model.
-#' @param create_chat_func A function that takes a system prompt and returns a
-#'   chat object. The default uses `ellmer::chat_openai()`.
+#' @param client An `ellmer::Chat` object, a string to be passed to
+#'   [ellmer::chat()] describing the model to use (e.g. `"openai/gpt-4o"`), or
+#'   a function that creates a chat client. If not provided, querychat consults
```

Comment:
Do we think it's actually useful to also support a function as an argument? FWIW, since there hasn't been a CRAN release, I wouldn't be entirely opposed to not supporting this and dropping `create_chat_func` entirely. If we do, let's just make sure to call it out in the `NEWS.md`